packages/apps/kubernetes/templates/oidc-authn-config.yaml

```
@@ -0,0 +1,87 @@
+    Either inline (oidc.customConfig.config — a string) or via a Secret
+    they have already created in the tenant namespace
+    (oidc.customConfig.secretRef.name). The chart writes the inline path
+    verbatim; for secretRef nothing is emitted here (cluster.yaml mounts
+    the user's Secret directly). Exactly one of the two MUST be set;
+    the chart fails the render otherwise.
+
+  None mode renders nothing.
+*/}}
+{{- if eq .Values.oidc.mode "System" }}
+{{-   $issuer := include "kubernetes.oidc.systemIssuerURL" . | trim }}
+{{-   $audience := include "kubernetes.oidc.clientId" . | trim }}
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)